hvplot/converter.py

```diff
@@ -459,53 +459,66 @@ class HoloViewsConverter:
         'global_extent',
     ]
 
-    _axis_options = [
-        'width',
+    _size_layout_options = [
+        'clim',
+        'fontscale',
+        'frame_height',
+        'frame_width',
         'height',
-        'shared_axes',
-        'grid',
-        'legend',
-        'rot',
-        'xlim',
-        'ylim',
-        'xticks',
-        'yticks',
-        'colorbar',
+        'max_height',
+        'max_width',
+        'min_height',
+        'min_width',
+        'padding',
+        'responsive',
+        'width',
+    ]
+
+    _axis_config_options = [
+        'clabel',
+        'flip_xaxis',
+        'flip_yaxis',
         'invert',
-        'title',
+        'loglog',
         'logx',
         'logy',
         'logz',
-        'loglog',
+        'rot',
+        'shared_axes',
+        'subcoordinate_y',
+        'title',
         'xaxis',
         'yaxis',
         'xformatter',
         'yformatter',
         'xlabel',
         'ylabel',
-        'clabel',
-        'padding',
-        'responsive',
-        'max_height',
-        'max_width',
-        'min_height',
-        'min_width',
-        'frame_height',
-        'frame_width',
-        'fontscale',
-        'bgcolor',
-        'subcoordinate_y',
-        'clim',
-        'flip_xaxis',
-        'flip_yaxis',
-        'tools',
+        'xlim',
+        'ylim',
+        'xticks',
+        'yticks',
+    ]
+
+    _grid_legend_options = [
+        'colorbar',
+        'grid',
+        'legend',
+    ]
+
+    _interactivity_options = [
         'hover',
         'hover_cols',
         'hover_formatters',
         'hover_tooltips',
+        'tools',
     ]
 
+    _axis_options = (
+        _size_layout_options + _axis_config_options + _grid_legend_options + _interactivity_options
```

Comment:
This formatting is entirely a ruff insistence via the pre-commit hooks. Not my choice!

Reply:
It looks okay to me! Tbh I think you should just remove `_axis_options` entirely. There are just a couple of places in the code where you'd have to replace it by the new ones you created.